python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -561,6 +562,20 @@ def makeGraph(self, pipeline, args):
         if args.qgraph_dot:
             graph2dot(qgraph, args.qgraph_dot)
 
+        if args.execution_butler_location:
+            butler = Butler(args.butler_config)
+            newArgs = copy.deepcopy(args)
+
+            def builderShim(butler):
+                newArgs.butler_config = butler._config
+                # This is done for all the side effects in makeWriteButler
```

Comment:
I don't really understand the comment. You are calling makeWriteButler so I'm not sure what "This" is you are referring to.